pyspark-notebook/Dockerfile

```diff
@@ -14,6 +14,15 @@ RUN apt-get -y update && \
     apt-get install --no-install-recommends -y openjdk-8-jre-headless ca-certificates-java && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
+
+# Install pyarrow
+RUN conda install --quiet -y 'pyarrow' && \
+    rm -rf $CONDA_DIR/share/jupyter/lab/staging && \
+    rm -rf /home/$NB_USER/.cache/yarn && \
+    rm -rf /home/$NB_USER/.node-gyp && \
```

Comment:
It's safe to remove the three `rm -rf` lines above. In the scipy image, we install some nodejs dependencies for jupyterlab and need to clean up working directories. pyarrow should be a pure Python package.

After this change, it LGTM.